### ISSUE TITLE
Should the vacuum also delete the other vacuum pods?

### ISSUE THREAD

**Issue by radu-matei** [Azure/brigade#846]
Are there any downsides if the vacuum would also delete other completed vacuum pods?

**Comment by radu-matei** [comment]
Should have actually checked before opening the issue. Closing.

**Comment by dgkanatsios** [comment]
What I usually do is let Kubernetes handle it by `kubectl edit cronjob brigade-brigade-vacuum` and set the `spec.successfulJobsHistoryLimit` to 2. In this way, I only have 2 vacuum Pods on a `Completed` state.

*or do you mean something completely different?*

**Comment by radu-matei** [comment]
Yeah, but ideally, users shouldn't have to edit the job after creation, but rather have this as an option when deploying the chart.

**Comment by radu-matei** [comment]
The default seems to be 10 - https://github.com/brigadecore/charts/blob/8708d0dfc8f46e2705516b866a85e7e76464d57a/charts/brigade/templates/vacuum-cronjob.yaml#L15

Should we _significantly_ lower that value?
